src/main/java/org/commonjava/indy/service/archive/jaxrs/ArchiveManageResources.java

```diff
@@ -109,88 +105,100 @@ public Response create( final @Context UriInfo uriInfo, final @Context HttpReque
             Optional<File> archive = controller.generateArchive( content );
             if ( archive.isEmpty() )
             {
-                final String message = "Failed to get downloaded contents for archive.";
-                logger.error( message );
-                return responseHelper.fromResponse( message );
+                return sendError( 500, "Failed to get downloaded contents for archive." );
```

Comment:
Not recommend to use magical number. Maybe Status.INTERNAL_SERVER_ERROR.getStatusCode() here and following?

Reply:
I've switched to sendResponse(Status status, String message), we don't have to use int to represent status code anymore